imgreg/sitk_multi_spect_registration.py

```diff
@@ -267,6 +274,7 @@ def align_channel(
         channel_params,
         img_shape,
         seed,
+        skip_metric_evaluate=True,
```

Comment:
Can we avoid a few hardcoded `True` by replacing this with `skip_metric_evaluate=skip_metric_evaluate`?

Reply:
I don't think this is possible in function definitions because the skip_metric_evaluate variable would be undefined.